src/main/java/Entities/GameLogicTree.java

```diff
@@ -0,0 +1,59 @@
+package Entities;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameLogicTree extends MenuTree{
+    private boolean didRoll;
+
+    public void setIsSwitchBlock(boolean switchBlock) {
+        this.switchBlock = switchBlock;
+    }
+
+    private boolean switchBlock = false;
```

Comment:
can we reorder to have instance attributes together

Reply:
these are stylistic but I think we can consider having the setter below the constructor

Reply:
I agree, we should refactor later to group up the instance attributes to make the code easier to read, and group up the getters and setters as well

Reply:
Yeah pretty difficult considering we might have to remove specific players in case of bankruptcy.

Reply:
I may be misunderstanding here but how would reordering the code affects the ability to remove players?

Reply:
These are just minor changes so yeah we can just reorder everything.